<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FractionCalculator</class>
 <widget class="QMainWindow" name="FractionCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FractionCalculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">FractionCalculator{
	background-color: #C3B1E1;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="num1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>171</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="num2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>171</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="denom1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="denom2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="selectedOperator">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>261</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="equals">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="answer">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fraction Calculator by Aidan Wolfson - 18AAW1 - 20147339</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>490</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Application</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
